tool/live/labd.sh

```
@@ -0,0 +1,58 @@
+  # daemon never creates login.keyring, and the first flutter_secure_storage
+  # write pops a gcr prompt that blocks the app main isolate forever — which
+  # looks exactly like a hung debugDumpApp.
+  eval "$(printf "\n" | gnome-keyring-daemon --unlock --components=secrets 2>/dev/null)"
+  export GNOME_KEYRING_CONTROL SSH_AUTH_SOCK
+  secret-tool store --label=prysmlab-probe prysmlab probe </dev/null 2>/dev/null \
+    && echo "PRYSMLAB_KEYRING_UP" || echo "PRYSMLAB_KEYRING_DEGRADED"
+  cd /work
+  # flutter run treats stdin EOF as quit, so hold stdin open with a writer that
+  # never writes.
+  sleep 2147483647 | flutter run -d linux --debug --no-pub --disable-service-auth-codes
+'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)